resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/VolcanoFeatureStep.scala

```diff
@@ -60,7 +59,6 @@ private[spark] class VolcanoFeatureStep extends KubernetesDriverCustomFeatureCon
 
     var spec = pg.getSpec
     if (spec == null) spec = new PodGroupSpec
```

Comment:
Hmm, when `spec = new PodGroupSpec`, what queue it will use?

Reply:
AFAIU it the `queue` should be in the podgroup template yaml

Reply:
I think this is for the case no podgroup template is specified?

Reply:
You are right! 
If `val pg = new PodGroup()` (line 53) then the PG won't have a queue

Reply:
To @viirya , yes. Volcano has a default queue by default and use it for the PodGroups without queue spec. That's the Volcano's pre-defined behavior.
- https://volcano.sh/en/docs/queue/#default-queue

To @martin-g , **no** as you see in the above Volcano document. Where did you get that understanding?
> AFAIU it the queue should be in the podgroup template yaml

Reply:
https://volcano.sh/en/docs/queue/#default-queue says `Subsequent jobs that are not assigned to a queue will be assigned to queue default.`. It is not clear whether the same applies to pod groups.
https://volcano.sh/en/docs/podgroup/ does not say anything explicitly too

Reply:
I just asked in Volcano Slack and they confirmed that the `default` queue will be used in this case.

Reply:
Yep, the volcano will create a default queue and also use [default queue internal](https://github.com/volcano-sh/volcano/blob/167e048eace4c7fcd192f8167359e9ad97c54545/pkg/webhooks/admission/jobs/mutate/mutate_job.go#L136) if spec.queue not specified.